eureka2-client/src/main/java/com/netflix/eureka2/client/resolver/EurekaRemoteResolverStep.java

```diff
@@ -0,0 +1,23 @@
+package com.netflix.eureka2.client.resolver;
+
+import com.netflix.eureka2.interests.Interest;
+import com.netflix.eureka2.registry.instance.InstanceInfo;
+
+/**
+ * @author David Liu
+ */
+public interface EurekaRemoteResolverStep {
```

Comment:
Can we add ServerResolverStep marker interface, which will connect all the step implementations into single rooted hierarchy?